service/machinelearning/internal/customizations/predictendpoint.go

```diff
@@ -0,0 +1,58 @@
+package customizations
```

Comment:
Nit it would be helpful to have a doc.go with package godoc explaining this customization and why it is here.